integration/bootstrap_networks/production/flannel.conf

```diff
@@ -0,0 +1,7 @@
+{
+  "cniVersion": "0.3.1",
+  "type": "flannel",
+  "delegate": {
+    "isDefaultGateway": true
```

Comment:
I have a vague memory of Flannel not properly installing default routes if "hairpinMode": true was not defined, but maybe it is not true anymore
can include it anyway, just in-case

Reply:
Done.

Reply:
Done.